checksums.json

```diff
@@ -1,147 +0,0 @@
-{
```

Comment:
When you build a puppet module the resulting .tar.gz file includes a generated checksums.json that contains the checksums of all files in the module. The file doesn't need to be in the repository as it's dynamically generated. This file hadn't been touched since the very first release - I think it got added by mistake at that point - so I don't think we need to add it to .gitignore unless it actually shows up again.